src/metabase/api/activity.clj

```diff
@@ -61,9 +62,11 @@
 (defendpoint GET "/"
   "Get recent activity."
   []
-  (filter mi/can-read? (-> (db/select Activity, {:order-by [[:timestamp :desc]], :limit 40})
-                               (hydrate :user :table :database)
-                               add-model-exists-info)))
+  (filter mi/can-read? (-> (db/select Activity {:order-by [[:timestamp :desc]]
+                                                :limit    offset-paging/*limit*
+                                                :offset   offset-paging/*offset*})
+                           (hydrate :user :table :database)
+                           add-model-exists-info)))
```

Comment:
added this because with the hardcoded limit of 40 it was possible the tests would never find the item they were looking for if we had > 40 activities already in the DB